server/zoom/meeting.go

```diff
@@ -3,12 +3,7 @@
 
 package zoom
 
-import (
-	"fmt"
-	"net/http"
-)
-
-// The User object defined at https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meeting.
+// https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meeting
```

Comment:
Not sure why removing the explanation. Nevertheless, to be more inline with GoDoc, I would put something like:
`// Meeting is defined at ...`